openweathermap-ios/openweathermap-ios/BookmarkLocation/Manager/BookmarkLocationManager.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  BookmarkLocationService.swift
+//  openweathermap-ios
+//
+//  Created by Cassio Sousa on 24/04/20.
+//
+
+import Foundation
+
+class BookmarkLocationManager: NSObject {
```

Comment:
Why are you inheriting from **NSObject**?

Reply:
Fix in b6bb083